portfolio/src/main/webapp/index.html

```diff
@@ -48,13 +48,21 @@ <h2>Academic Achievements</h2>
     </ul>
 
     <p> View my transcript <a href="images/AcademicTranscript.pdf">here</a></p>
-    <button onclick="getFromServlet()">Click me!</button>
-    <ul id="comment-container"></ul>
     <br>
+    <h3>Give feedback!</h3>
     <form action="data" method="POST">
         <input type="text" name="user-comment">
         <input type="submit"/>
     </form>
+    <br>
+    <form>
+    <p> Number of comments to show</p>
+    <input type = "number" id="load-comments" onchange="getFromServlet(this.value)" value ="0" max="10" min="0"> 
```

Comment:
Correct me if I'm wrong, but the comments don't show until this is changed right?  One way to fix this would be to add an onload event handler to your \<body\> element

Reply:
Yes, you are correct. The value is set to 0 at the beginning so no comments show at first. I sorta like that there are no comments until select a number. Do you think comments showing at the beginning would be better?

Reply:
Well I think even if you set the value to non-zero, you would still see no comments at first.  But if you prefer that behavior this is fine